src/main/scala/org/bitcoins/core/gen/ScriptGenerators.scala

```diff
@@ -177,8 +223,15 @@ trait ScriptGenerators extends BitcoinSLogger {
   } yield {
     val publicKeys = privateKeys.map(_.publicKey)
     val scriptPubKey = MultiSignatureScriptPubKey(requiredSigs,publicKeys)
+    val (signedScriptSig, _, _) = multiSigHelper(privateKeys, requiredSigs, scriptPubKey)
+    (signedScriptSig, MultiSignatureScriptPubKey(scriptPubKey.hex), privateKeys)
```

Comment:
`MultiSignatureScriptPubKey(scriptPubKey.hex)` is unnecessary, you can just do `scriptPubKey`